optaplanner-core/src/main/java/org/optaplanner/core/impl/partitionedsearch/queue/PartitionQueue.java

```diff
@@ -31,7 +31,7 @@
 import org.slf4j.LoggerFactory;
 
 /**
- * Thread-safe
+ * Thread-safe.
```

Comment:
Looks like I had several other instances with a non-sentance that ended with a dot in javadocs. Fixed.